Dockerfile-cyclictest

```diff
@@ -2,7 +2,7 @@ FROM centos:8
 USER root
 COPY cyclictest/cmd.sh /root
 COPY common-libs /root/common-libs
-RUN RT_TEST=$(curl -L https://www.rpmfind.net/linux/centos/8-stream/AppStream/x86_64/os/Packages/ 2>/dev/null | sed -n -r 's/.*href=\"(rt-tests.*.rpm).*/\1/p') \
+RUN RT_TEST=$(curl -L https://www.rpmfind.net/linux/centos/8-stream/AppStream/x86_64/os/Packages/ 2>/dev/null | sed -n -r 's/.*href=\"(rt-tests-2.1-2.*.rpm).*/\1/p') \
```

Comment:
do we want to pin to this particular version? alternatively we can use "sed -n -r 's/.*href=\"(rt-tests.*.rpm).*/\1/p') | tail -1"  to get the last entry without pin.

Reply:
Good question - I wasn't sure that the list of RPMs returned would always be in the right order. If that is the case, then I'm OK with changing this to use the last one in the list. Let me know what you prefer - I think it is OK either way.